index.html

```diff
@@ -0,0 +1,178 @@
+</head>
+
+<body>
+    <header>
+        <div id="blackHeader">
+            
+            <div class="blackIconsContainer">
+                <a href=""><img class="blackHeaderIcon" src="./images/facebookIcon.png" alt=""></a>
+                <a href=""><img class="blackHeaderIcon" src="./images/twitterIcon.png" alt=""></a>
+            </div>
+            <a href="">
+                <div class="blackHeaderOption">English</div>
+            </a>
+            <a href="">
+                <div class="blackHeaderOption">My Page</div>
+            </a>
+            <a href="">
+                <div class="blackHeaderOption">Logout</div>
+            </a>
+
+        </div>
+        <div id="whiteHeader">
+            <a class="logoLink" href="./index.html"><img id="logoHeader" src="./images/logoOrangeMx-removebg-preview.png" alt="logo"></a>
+            <div id="menuOptions">
+                <a class="menuOptionsLinks" href="./about.html">
+                    <div>About</div>
+                </a>
+                <a class="menuOptionsLinks" href="./about.html">
+                    <div>Event</div>
+                </a>
+                <a class="menuOptionsLinks" href="#programDetails">
+                    <div>Join</div>
+                </a>
+                <a class="menuOptionsLinks sponsorMobile" href="./about.html#footerOne">
+                    <div>Sponsor</div>
+                </a>
+                <a class="menuOptionsLinks sponsorDesk" href="#footerOne">
+                    <div>Sponsor</div>
+                </a>
+                <a class="menuOptionsLinks" href="#">
+                    <div>News</div>
+                </a>
+                <a class="menuOptionsLinks ccCampaign" href="#">
+                    <div>CC Campaign</div>
+                </a>
+            </div>
+        </div>
+    </header>
+    <main>
+        <!-- Main Section -->
+        <section class="main">
+                <div id="mobileTopContainer">
+                    <nav class="hamburgerMenu">
+                        <span class="bar bar1"></span>
+                        <span class="bar bar2"></span>
+                        <span class="bar bar3"></span>
+                    </nav>
+                    <div id="firstMain"></div>
+                </div>
+                <!-- ---- -->
+                <img class="adornIcon taco" src="./images/iconsMxImages/6bigTaco-removebg-preview.png" alt="">
+                <img class="adornIcon hat" src="./images/iconsMxImages/2hat-removebg-preview.png" alt="">
+                <img class="adornIcon pinata" src="./images/iconsMxImages/3pin-removebg-preview.png" alt="">
+                <img class="adornIcon guacamole" src="./images/iconsMxImages/7guacamole-removebg-preview.png" alt="">
+                <!-- --- -->
+                <div id="textMain">
+                    <div id="helloAndEventContainer">
+                        <div id="hello">"Hello! Food Lovers"</div>
```

Comment:
- [Optional] kindly use an h3/h4 tag here for semantic purposes. Remember it is a subheading